mii/client.py

```diff
@@ -35,12 +52,23 @@ def mii_query_handle(deployment_name):
         query_handle: A query handle with a single method `.query(request_dictionary)` using which queries can be sent to the model.
     """
 
-    if deployment_name in mii.non_persistent_models:
-        inference_pipeline, task = mii.non_persistent_models[deployment_name]
-        return MIINonPersistentClient(task, deployment_name)
+    if deployment_tag in mii.non_persistent_models:
+        inference_pipeline, task = mii.non_persistent_models[deployment_tag]
+        return MIINonPersistentClient(task, deployment_tag)
+
+    deployments, lb_config, model_path, port_map = _get_deployment_configs(deployment_tag)
+    mii_configs = None
+    if len(deployments) > 0:
+        mii_configs = next(iter(deployments.values())).mii_config
```

Comment:
Does this assume that all the `mii_config` have the same port number value? Perhaps we need to rethink what the port number field in the `MIIConfig` is meant for with support for multi-model deployments

Reply:
Add new check to ensure that all the port numbers are the same